### PR TITLE
migration to add fields to orders and change orders def in swagger

### PR DESCRIPTION
**Pull request by kilbergr**
## Description
This PR adds a get and patch endpoint for orders model, which has also been updated via migrations to include 3 new fields:
 * Current duty station
 * Orders type detail
 * Orders number
These are all optional fields, because they will be used by the office after order model creation by service member. 

## Reviewer Notes

Current duty station is optional. In the model, I've included as follows:
```
CurrentDutyStationID   *uuid.UUID                         `json:"current_duty_station_id" db:"current_duty_station_id"`
	CurrentDutyStation     *DutyStation                        `belongs_to:"duty_stations"`
```

Is that correct? Wasn't sure how to use the pointer in this circumstance.

Also--figuring our now how to update sqs diagram. 

## Code Review Verification Steps

* [x] All tests pass.
* [ ] Any migrations/schema changes also update the diagram in docs/schema/dp3.sqs
* [x] Request review from a member of a different team.
* [x] (TRIAL) Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](https://www.pivotaltracker.com/story/show/157373169) for this change

### CODE REVIEW

File: swagger/internal.yaml [hunk 1]
```diff
@@ -1141,7 +1141,26 @@ definitions:
       DEPENDENT_TRAVEL: Dependent Travel
       BLUEBARK: Bluebark
       VARIOUS: Various
-  OrdersPayload:
+  OrdersTypeDetail:
+    type: string
+    title: Orders type details
+    enum:
+      - hhg_permitted
```
Comment (macrael) [inline swagger/internal.yaml]: Please make these ALL_CAPS like our other enums.

File: swagger/internal.yaml [hunk 2]
```diff
@@ -1166,13 +1185,24 @@ definitions:
...
         $ref: "#/definitions/OrdersType"
+      orders_type_detail:
+        $ref: "#/definitions/OrdersTypeDetail"
+        x-nullable: true
       has_dependents:
         type: boolean
         title: Are dependents included in your orders?
       new_duty_station:
         $ref: '#/definitions/DutyStationPayload'
+      current_duty_station:
```
Comment (macrael) [inline swagger/internal.yaml]: Why is this attached to Orders? We already have this info on the Service Member, can that info conflict?

File: swagger/internal.yaml [hunk 3]
```diff
@@ -1916,6 +1957,68 @@ paths:
           description: move not found
         500:
           description: internal server error
+  /moves/{moveId}/orders:
```
Comment (macrael) [inline swagger/internal.yaml]: I don't think you need to create a new endpoint to get and patch orders, you will be able to use the one that already exists. 

your move with have an orders_id in it. then you can GET /orders/{orders_id} and PATCH or PUT it.

Comment (ntwyman) [reply swagger/internal.yaml]: I think GET /move/move_id/orders is a reasonable helper, but PATCH should be on /orders/orders_id.

File: swagger/internal.yaml (hunk 2 above)
Comment (ntwyman) [reply swagger/internal.yaml]: Orders might be in the future. You can have several outstanding sets of orders, e.g. first you got to washingto for 2 months for training then you go St Louis. Prolly better to suffer the duplication and keep things flexible.

Comment (ntwyman) [reply swagger/internal.yaml]: I do wonder why current-duty station is optional but new duty station is required.

What happens when you enlist or when you retire?

Comment (kilbergr) [reply swagger/internal.yaml]: That's a good point. I wonder whether new_duty_station should be optional too.

**Pull request merged**